ci-operator/step-registry/rosa/cluster/provision/rosa-cluster-provision-commands.sh

```diff
@@ -438,7 +438,7 @@ fi
 
 PRIVATE_LINK_SWITCH=""
 if [[ "$PRIVATE_LINK" == "true" ]]; then
-  PRIVATE_LINK_SWITCH="--private-link"
+  PRIVATE_LINK_SWITCH="--private-link --default-ingress-private"
```

Comment:
we should use `--private  --default-ingress-private`  according to @YiqinZhang 's suggestion